src/auth/screens/auth-profile.screen.js

```diff
@@ -143,6 +147,38 @@ class AuthProfile extends Component {
                 />
               )}
             </SectionList>
+
+            <SectionList title="Sign Out">
+              <ListItem
+                title="Sign Out"
+                titleStyle={styles.logoutTitle}
+                leftIcon={{
```

Comment:
So sorry for the back and forth here, but having the icon not aligned with the organization avatars above seem a little off. Let's try removing it so it looks like this:

![image](https://user-images.githubusercontent.com/12476932/28295751-10ec6c02-6b31-11e7-8155-6e923c45eada.png)

Similarly let's add a `hideChevron` prop to hide the right icon